lib/accesscontrol/permitted_actions/base.rb

```diff
@@ -0,0 +1,40 @@
+require "accesscontrol/query_builder"
+
+module AccessControl
+  module PermittedActions
```

Comment:
If we'd like to use `PermittedAction` then we'll need to rename our `PermittedAction` model or move that model inside another namespace

Reply:
I could see us going either way. Up to you!